### PR TITLE
Contributed for hacktoberfest

### PR DESCRIPTION
**Pull request by komangss**
Happy hacktoberfest! :)

### CODE REVIEW

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@komangss, Kindly add 3 resources that can help developers with title, link and name. Thanks!

**Comment by TenTraicion** [comment]
@komangss, Kindly follow our instruction given in our project [README.md](https://github.com/Syknapse/Contribute-To-This-Project#contribute) properly.

File: index.html [hunk 1]
```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
...
+            <p class="contact">
+              <i class="fab fa-twitter"></i>
+              <a href="https://twitter.com/KomangAntonio" target="_blank">@KomangAntonio</a>
+            </p>
+            <p class="about">A person that is searching for a passion</p>
+            <div class="resources">
+              <p>3 Useful Dev Resources</p>
+              <ul>
+                <li>
+                  <a href="#" target="_blank" title="First Resource">Resource 1</a>
```
Comment (Syknapse) [inline index.html:130]: please provide links here in the 3 resources section

**Review by Syknapse — CHANGES_REQUESTED** [review]

**Comment by TenTraicion** [comment]
@komangss closing this.
Reason: No response. You can always make the change and reopen or create a new PR. Thanks!

for any kind of query you can ask here or in our [Discord Community: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF)

**Pull request closed without merging**